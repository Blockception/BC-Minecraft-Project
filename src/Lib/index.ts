/*	Auto generated	*/

export * from "./mcattributes";
export * from "./mcdefinitions";
export * from "./mcignore";
export * from "./project";