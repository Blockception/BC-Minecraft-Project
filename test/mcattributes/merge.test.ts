import { MCAttributes } from '../../src/main';

describe("MCAttributes", () => {
  it("given two mcattributes, can be merged", () => {
    const first: MCAttributes = {
      "property-1": "value-1",
      "property-2": "value-1",
      "property-3": "value-1",
      "property-4": "value-1",
      "property-5": "value-1",
      "property-6": "value-1",
    };
    const second: MCAttributes = {
      "property-5": "value-2",
      "property-6": "value-2",
      "property-7": "value-2",
      "property-8": "value-2",
      "property-9": "value-2",
    };

    const merged = MCAttributes.merge(first, second);
    expect(merged).toMatchSnapshot()
  })
})