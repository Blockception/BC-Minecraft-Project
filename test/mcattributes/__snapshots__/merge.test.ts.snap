// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`MCAttributes given two mcattributes, can be merged 1`] = `
{
  "property-1": "value-1",
  "property-2": "value-1",
  "property-3": "value-1",
  "property-4": "value-1",
  "property-5": "value-2",
  "property-6": "value-2",
  "property-7": "value-2",
  "property-8": "value-2",
  "property-9": "value-2",
}
`;
